plasma_framework/test/src/exits/OutputGuardParserRegistry.test.js

```diff
@@ -0,0 +1,66 @@
+const OutputGuardParserRegistry = artifacts.require('OutputGuardParserRegistry');
+const DummyOutputGuardParser = artifacts.require('DummyOutputGuardParser');
+
+const { constants, expectRevert } = require('openzeppelin-test-helpers');
+const { expect } = require('chai');
+
+contract('OutputGuardParserRegistry', ([_, other]) => {
+    beforeEach(async () => {
+        this.dummyOutputGuardParser = await DummyOutputGuardParser.new(constants.ZERO_ADDRESS);
+        this.registry = await OutputGuardParserRegistry.new();
+    });
+
+    describe('outputGuardParsers', () => {
+        it('should receive empty outputGuardParser contract address when output type not registered', async () => {
+            const nonRegisteredOutputType = 66666;
+            expect(await this.registry.outputGuardParsers(nonRegisteredOutputType)).to.equal(constants.ZERO_ADDRESS);
+        });
+    });
+
+    describe('registerOutputGuardParser', () => {
+        it('should be able to register successfully', async () => {
+            const outputType = 1;
+            await this.registry.registerOutputGuardParser(outputType, this.dummyOutputGuardParser.address);
+            expect(await this.registry.outputGuardParsers(outputType)).to.equal(this.dummyOutputGuardParser.address);
+        });
+
+        it('should reject when not registered by operator', async () => {
+            await expectRevert(
+                this.registry.registerOutputGuardParser(1, this.dummyOutputGuardParser.address, { from: other }),
+                'Not being called by operator',
+            );
+        });
+
+        it('should reject when frozen', async () => {
```

Comment:
I think currently anyone can freeze. Don't we want to limit that?

Reply:
oh yeah, I missed that. However, I just realized openzepplin `Ownable` contract has the function of [renounceOwnership(...)](https://github.com/OpenZeppelin/openzeppelin-solidity/blob/master/contracts/ownership/Ownable.sol#L54) which can do the same thing as we want.